mods/cnc/maps/gdi01/map.yaml

```diff
@@ -424,9 +424,6 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		-CrateSpawner:
-		PlayMusicOnMapLoad:
```

Comment:
Do we need an upgrade rule for this? I guess we can't rule out that some maps on r.o.n. uses it, although I'd say it's unlikely.